keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1724,6 +1730,14 @@ Pod scheduling and startup are more important. However, expected performance
 will depend on how resources are used (for example, how often new Pods are
 created), therefore it is impossible to predict what reasonable SLOs might be.
 
+There shouldn't be a significant performance impact on the Kubelet side as
+Resource manager component will do its work similarly to the Volume manager:
```

Comment:
Arguing that it will have little impact because it is similar to some other component seems a bit optimistic to me :sweat_smile: 

That other component might have significant impact, and then so will we. I would rephrase this:
```suggestion
Resource manager component will do its work similarly to the Volume manager, but the overhead and complexity should be lower:
```

Reply:
The similarity to Volume manager is not the only argument. It's just a start of the explanation. It simply refers to the fact that both upper level components are run as goroutines, similarly to Volume manager.

Reply:
But it is not a valid argument... similarity doesn't imply anything regarding performance impact.

Reply:
It does in my opinion. If it would do it sequentially it can slow down the Kubelet.

Reply:
That it uses goroutines is a valid argument.

Let's just simplify the statements, then it will become clearer, okay? Start by stating that the resource manager will be similar to the volume manager. Describe how it will be simpler. Then conclude with an estimate of the the performance overhead.

Reply:
Also beware that "performance" could mean many things. If we consider the time required to start a pod, then the resource manager is on the critical path and running its code in goroutines doesn't change anything.

Reply:
Claiming "no significant performance impact" without being able to provide benchmark results that back up such a claim is risky. Reviewers will jump on this and ask for proof.

Reply:
> Can you suggest how to put it better?